multivac/sensors/test_status.py

```diff
@@ -58,15 +61,27 @@ def test_status_iter(log_fh, cache_filepath=None):
         cache = []
 
     hang_detected = False
+
+    saved_data = {}
```

Comment:
`saved_data` is name of a variable that does not define its contents.
It is also not a good practice to have `data/dict/list` in the var name.

Reply:
fixed